### DISCUSSION TITLE
How to create a default shared folder

### DISCUSSION THREAD

**Discussion by am0ral93** [drakkan/sftpgo#811] [General]
Hey! Can you please tell me how to create a default folder without a password? Make an entrance to FTP (it is possible without the web version) by going to the address without a login and password?

I apologize in advance if this question has already been asked or is in the manual, but I did not find it unfortunately

**Comment by drakkan** [comment]
Hi,

anonymous accounts are not explicitly supported, some workarounds have been discussed in #373.

If anyone wants to support this development with a small sponsorship I can add it, thanks

**Comment by am0ral93** [comment]
Thanks for the answer! The project is a miracle, I'm surprised that there is no such function, after so much time from the start of management. For our project, this is very bad, well, let's wait.

**Reply by drakkan** [reply]
> Thanks for the answer! The project is a miracle, I'm surprised that there is no such function, after so much time from the start of management. For our project, this is very bad, well, let's wait.

If you wait passively this will never happen :-)
.
Maintaining an open source project like this takes a lot of time and effort and I don't even use it in production myself. I don't think you work for free :smile:

Companies should learn to support open source projects, not just use them and ask for support and new features for free

**Reply by am0ral93** [reply]
I would gladly transfer to you from my own funds, but not from the company's funds. Yes, and I think whoever needs it will support it without it.
